src/dotgov.scss

```diff
@@ -81,6 +81,7 @@
 @import "./styles/partials/components/content";
 @import "./styles/partials/components/explore";
 @import "./styles/partials/components/scrollable-container";
+@import "./styles/partials/components/progresstabs";
```

Comment:
can you change the name of this file to `progress-tabs` to match the class name?

Reply:
done